griptape/drivers/sql/amazon_redshift_sql_driver.py

```diff
@@ -1,14 +1,17 @@
+from __future__ import annotations
```

Comment:
Is this imported for side effects, unused, or is there some other magic happening here?

Reply:
Yeah it's some python magic that lets us use the `TYPE_CHECKING` functionality even thought it's not technically released in our current python version. You can read more about it [here](https://stackoverflow.com/questions/61544854/from-future-import-annotations).